helfi_ahjo.module

```diff
@@ -97,3 +97,22 @@ function helfi_ahjo_preprocess_paragraph(&$variables) {
     $variables['#cache']['tags'][] = 'taxonomy_term_list:sote_section';
   }
 }
+
+/**
+ * Implements hook_config_ignore_settings_alter().
+ */
+function helfi_ahjo_config_ignore_settings_alter(array &$settings) {
+  // Chatleijuke is used in hammashoito chat in Sote.
+  // Genesyschat might be used in palvelukeskus later but
+  // the block configs already removed from other instances.
```

Comment:
This comment is not relevant for this module.

Reply:
I don't think we need to add this modules config to config ignore at all, since we should set api keys in `*settings.php`.